packages/cli/src/commands/bootstrap/bootstrap.command.ts

```diff
@@ -26,7 +26,7 @@ const handler: (options: Options) => Promise<void> = async () => {
   });
```

Comment:
This refactor is in a seperate commit:

![image](https://user-images.githubusercontent.com/793481/203535440-d9cfdecf-c254-46d3-90d7-ff20d9602d47.png)

So may be easier to review that commit, and then review the other commits